test/next/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Node",
```

Comment:
this doesn't change anything, just TS automatically set it with a capital Node somehow if you change it